packages/react-redux-subspace/docs/api/useSubspace.md

```diff
@@ -0,0 +1,125 @@
+# `useSubspace`
+
+A [React hook](https://reactjs.org/docs/hooks-custom.html#using-a-custom-hook) that creates a [subspace](/packages/redux-subspace/docs/api/subspace.md) for the current store context.
+
+## Arguments
+
+1. `mapState` (_Function|string_): A [selector to derive the state](/docs/basics/CreatingSubspaces.md) of the subspace. The selector is provided the parent state as the first parameter and the root state as the second parameter. If passed as a string, a selector is created for that key on the provided state.
+2. `namespace` (_string_): An optional [namespace to scope actions](/docs/basics/Namespacing.md) with.
+3. `options` (_Object_): An optional object to supply the following options:
+   - `context` (_React.Context|Object_): Override the React Context used for accessing the store.
+
+If `mapState` is passed as a string and no `namespace` is provided, the provided string is used for both. To prevent this, pass `null` as the second parameter.
+
+## Returns
+
+(_Object_): The subspaced store.
+
+## Examples
```

Comment:
You might want to name each of these examples or give some description of what you are showing in the example. as right now it might be a bit messy.